.github/workflows/build-deploy-pudl.yml

```diff
@@ -78,6 +78,10 @@ jobs:
       - name: Set up Cloud SDK
         uses: google-github-actions/setup-gcloud@v0
 
+      # Make sure the public and private zenodo-cache buckets are in sync
+      - name: Sync zenodo-cache buckets
+        run: gsutil -m rsync -d -r gs://internal-zenodo-cache.catalyst.coop gs://zenodo-cache.catalyst.coop
+
```

Comment:
The internal bucket is now the source, and the external bucket is the destination because the internal bucket will be updated more frequently with any new archives. Currently, the buckets have identical contents.

Reply:
What is the mechanism for updating the internal cache? Does it just happen by virtue of a build running and needing to get new data because we pushed code referring to a new Zenodo DOI?  If so, then the rsync will update the public bucket with whatever new data was downloaded by the previous day's build, right? (since the rsync runs before the build).

Reply:
Yes, the GCS cache is updated during a build running and needing to get new data because of a new zenodo DOI. Currently, this is the workflow:

1. Code with new zenodo DOI merged into dev.
2. Github action syncs zenodo caches (no changes)
3. The nightly build is run and updates the internal zenodo cache.
4. The following night, the github action syncs the zenodo caches (new changes)

I think it probably makes more sense for `gcp_pudl_etl.sh` to sync the buckets after the etl finishes so there isn't a delay. I can make the change today.

Reply:
Splitting the sync out to #1895